lightning_examples/reinforce-learning-DQN/dqn.py

```diff
@@ -0,0 +1,388 @@
+# -*- coding: utf-8 -*-
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.2
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %%
+import os
+from collections import deque, namedtuple, OrderedDict
+from typing import List, Tuple
+
+import gym
+import numpy as np
+import torch
+from pytorch_lightning import LightningModule, Trainer
+from torch import nn, Tensor
+from torch.optim import Adam, Optimizer
+from torch.utils.data import DataLoader
+from torch.utils.data.dataset import IterableDataset
+
+PATH_DATASETS = os.environ.get('PATH_DATASETS', '.')
+AVAIL_GPUS = min(1, torch.cuda.device_count())
+
+
+# %%
+class DQN(nn.Module):
+    """
+    Simple MLP network
+    """
+
+    def __init__(self, obs_size: int, n_actions: int, hidden_size: int = 128):
+        """
+        Args:
+            obs_size: observation/state size of the environment
+            n_actions: number of discrete actions available in the environment
+            hidden_size: size of hidden layers
+        """
```

Comment:
```suggestion
    """
    Simple MLP network
    
    Args:
        obs_size: observation/state size of the environment
        n_actions: number of discrete actions available in the environment
        hidden_size: size of hidden layers
    """

    def __init__(self, obs_size: int, n_actions: int, hidden_size: int = 128):
```

Reply:
is there a convention for this? I would tend to rather have the args explanation closer to the method itself...

Reply:
That would be fine, although then should be consistent and do that for all classes in the notebook haha 😆